test/subset/run-tests.py

```diff
@@ -104,7 +104,7 @@ def run_test (test, should_check_ots):
 
 def run_ttx (font_path, ttx_output_path):
 	print ("fonttools ttx %s" % font_path)
-	return cmd ([fonttools, "ttx", "-q", "-o", ttx_output_path, font_path])
+	ttx.main (args=['-q', '-o', ttx_output_path, font_path])
```

Comment:
Probably not much faster, but you can skip ttx altogeyher, e.g.:
```python
from fontTools.ttLib import TTFont
with TTFont(font_path) as font:
	font.saveXML(ttx_output_path)
```

Reply:
You need to also return something from this function in case of failure.

Reply:
Since the code is saving to a file, just to read it back, you can also skip this and pass a `file`-like object (e.g. `io.StringIO` ) and skip writing to disk.

Reply:
Thank you so much Khaled for the suggestions, have applied it on https://github.com/harfbuzz/harfbuzz/commit/322426f8c8ef818a60ec9090670ff077647fdfa2 please let me know anything else can be improved. Thanks! :)